src/js/components/stock-movement-wizard/outbound/AddItemsPage.jsx

```diff
@@ -70,21 +70,30 @@ const NO_STOCKLIST_FIELDS = {
     isFirstPageLoaded: ({ isFirstPageLoaded }) => isFirstPageLoaded,
     addButton: ({
       // eslint-disable-next-line react/prop-types
-      addRow, getSortOrder, showOnly, updateTotalCount,
+      addRow, getSortOrder, showOnly, updateTotalCount, getStockMovementDraft, isDraftAvailable,
     }) => (
-      <button
-        type="button"
-        id="addButton"
-        className="btn btn-outline-success btn-xs"
-        disabled={showOnly}
+      <>
```

Comment:
Have you made sure it is working without having to write
```javascript
 <React.Fragment>
```
I remember we had to do it like this, since `<>` didn't work, but probably after I bumped up ESLint version, it might be working now, so confirm please

Reply:
Yes, it's working now